src/app/personside/infotabs/oppfolging/OppfolgingDatoKomponent.tsx

```diff
@@ -52,38 +50,40 @@ interface DispatchProps {
 
 type Props = DispatchProps & StateProps;
 
-const tidligsteDato = () =>
-    moment()
-        .subtract(10, 'year')
-        .startOf('day')
-        .toDate();
+const tidligsteDato = () => {
+    const dato = new Date();
+    dato.setFullYear(dato.getFullYear() - 10);
+    return dato.toLocaleDateString('en-CA');
```

Comment:
Syns ikke vi skal basere oss på engelsk-kanadisk formattering "bare for å få det til å fungere".